zerver/actions/streams.py

```diff
@@ -145,21 +143,7 @@ def do_deactivate_stream(stream: Stream, *, acting_user: Optional[UserProfile])
     # changing stream privacy. And due to this we also need to duplicate
     # the code to unset is_web_public field on attachments below.
     stream.deactivated = True
-    stream.invite_only = True
-    # Preserve as much as possible the original stream name while giving it a
-    # special prefix that both indicates that the stream is deactivated and
-    # frees up the original name for reuse.
-    old_name = stream.name
-
-    # Prepend a substring of the hashed stream ID to the new stream name
-    streamID = str(stream.id)
-    stream_id_hash_object = hashlib.sha512(streamID.encode())
-    hashed_stream_id = stream_id_hash_object.hexdigest()[0:7]
-
-    new_name = (hashed_stream_id + "!DEACTIVATED:" + old_name)[: Stream.MAX_NAME_LENGTH]
-
-    stream.name = new_name[: Stream.MAX_NAME_LENGTH]
-    stream.save(update_fields=["name", "deactivated", "invite_only"])
```

Comment:
The commit message needs to be very clear that this also no longer marks them as `invite_only`.

Reply:
Okay so I undo my changes. I think a second PR for this change will be better.